tap_hubspot/schemas/contact_list.json

```diff
@@ -0,0 +1,257 @@
+          }
+        }
+      }
+    },
+    "merge-audits": {
+      "type": ["null", "array"],
+      "items": {
+        "type": ["null", "object"],
+        "properties": {
+          "canonical-vid": {
+            "type": ["null", "integer"]
+          },
+          "vid-to-merge": {
+            "type": ["null", "integer"]
+          },
+          "timestamp": {
+            "type": ["null", "string"],
+            "format": "date-time"
+          },
+          "user-id": {
+            "type": ["null", "integer"]
+          },
+          "num-properties-moved": {
+            "type": ["null", "integer"]
+          },
+          "merged_from_email": {
+            "type": ["null", "object"],
+            "properties": {
+              "value": {
+                "type": ["null", "string"]
+              },
+              "source-type": {
+                "type": ["null", "string"]
+              },
+              "source-id": {
+                "type": ["null", "string"]
+              },
+              "source-label": {
+                "type": ["null", "string"]
+              },
+              "source-vids": {
+                "type": ["null", "array"],
+                "items": {
+                  "type": ["null", "integer"]
+                }
+              },
+              "timestamp": {
+                "type": ["null", "integer"]
+              },
+              "selected": {
+                "type": ["null", "boolean"]
+              }
+            }
+          },
+          "merged_to_email": {
+            "type": ["null", "object"],
+            "properties": {
+              "value": {
+                "type": ["null", "string"]
+              },
+              "source-type": {
+                "type": ["null", "string"]
+              },
+              "source-id": {
+                "type": ["null", "string"]
+              },
+              "source-label": {
+                "type": ["null", "string"]
+              },
+              "timestamp": {
+                "type": ["null", "integer"]
+              },
+              "selected": {
+                "type": ["null", "boolean"]
+              }
+            }
+          }
+        }
+      }
+    },
+    "properties": {
+      "type": "object",
+      "properties": {
+        "firstname": {
+          "type": "object",
+          "properties": {
+            "value": {
+              "type": "string"
+            }
+          },
+          "required": [
+            "value"
+          ]
+        },
+        "lastmodifieddate": {
+          "type": "object",
+          "properties": {
+            "value": {
+              "type": "string"
+            }
+          },
+          "required": [
+            "value"
+          ]
+        },
+        "company": {
+          "type": "object",
+          "properties": {
+            "value": {
+              "type": "string"
+            }
+          },
+          "required": [
+            "value"
```

Comment:
The trick is that if we have a `company` object in the result it will contain `value`.
Considering that HubSpot may suddenly change their stable API (as they did it once with `filter` property of a contact_lists entity it makes sense to remove these constraints. But from the other hand if they changed their API I'd rather be aware of that and adjust the schema then just let it slide. Your take?

Reply:
Do you mean if Company object is present value param is required, else Company object may be missing, and it runs fine?

Reply:
Yes, exactly

Reply:
Its good then.